pydynamo_brain/pydynamo_brain/ui/actions/radiiActions.py

```diff
@@ -24,7 +35,92 @@ def changeRadius(self, localIdx, dR, laterStacks=False):
             for atIndex in range(localIdx, upperBound):
                 current = self._localState(atIndex).currentPoint()
                 if current is not None:
+                    current.manuallyMarked = False
                     if current.radius == None:
                         current.radius = 10 * dR
                     else:
                         current.radius *= dR
+
+
```

Comment:
I'd recommend having these either in util, or as functions in the class
(e.g. def _radiiThreshold(self, ...))

it's uncommon to see non-class functions in class files.